jina/drivers/search.py

```diff
@@ -162,16 +161,31 @@ class SparseVectorSearchDriver(VectorSearchDriver):
     Extract sparse embeddings from the request for the executor to query.
     """
 
-    @staticmethod
-    def _get_documents_embeddings(docs: 'DocumentSet'):
-        return docs.all_sparse_embeddings
+    @property
+    def exec_sparse_cls_type(self) -> str:
+        """Get the sparse type from executor.
+
+        :return: Sparse matrix type, default value is `scipy_coo`.
```

Comment:
do not put this note here as it is hard to mantain. The default value will be in the base class of executor, not here